src/utils/string_utils.F90

```diff
@@ -4,6 +4,7 @@ module string_utils
    use shr_string_mod, only: to_lower => shr_string_toLower
    use cam_logfile,    only: iulog
    use cam_abortutils, only: endrun
+   use string_core_utils, only: to_str, int_date_to_yyyymmdd, int_seconds_to_hhmmss, stringify
```

Comment:
I can't remember if in Fortran this method allows `to_str` and `stringify` to be publicly accessible from this module without renaming the functions like what is done above with `to_upper` and `to_lower`.  It might be worth running the SIMA regression tests (once that PR is in) with these changes to make sure it works as expected (assuming you haven't run tests like that already).